src/citrine/resources/gemtables.py

```diff
@@ -285,7 +287,7 @@ def delete(self, uid: Union[UUID, str]):
         """Tables cannot be deleted at this time."""
         raise NotImplementedError("Tables cannot be deleted at this time.")
 
-    def read(self, table: Union[GemTable, Tuple[str, int]], local_path: str):
+    def read(self, table: Union[GemTable, Tuple[str, int]],  *, local_path: str):
```

Comment:
I don't have strong feelings about this one, but since both arguments are equally useful I err towards making them both be keyword args

Reply:
I was thinking of having this one and the `initiate_build` match the other methods here like `delete`, `update`, etc

Reply:
I agree on `initiate_build`. I suppose they're similar from the user's perspective. The difference I see is that `read` requires both arguments, while `initiate_build` generally requires only one. But it can go either way.

Related, this method is missing a full docstring (with parameters and a return value). Would you mind adding that?